Makefile.toml

```diff
@@ -654,6 +670,14 @@ if [[ $ENABLE_COREDUMP == "true" ]]; then
   ulimit -c unlimited
 fi
 
+if [[ -z "$RW_QUERY_LOG_PATH" ]]; then
+  export RW_QUERY_LOG_PATH="${PREFIX_LOG}"
+fi
+
+if [[ -z "$RUST_LOG" ]]; then
+  export RUST_LOG="pgwire_query_log=info"
```

Comment:
Can we always concatenate this directive instead of only setting it if absent?

Reply:
What if someone want to disable it? 🤣

Reply:
Oh, we may put it as the first one